app/api/events_orga.py

```diff
@@ -0,0 +1,65 @@
+from flask import request, current_app
+from flask_jwt import current_identity, _jwt_required
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from flask_rest_jsonapi.exceptions import ObjectNotFound
+from marshmallow_jsonapi import fields
+from marshmallow_jsonapi.flask import Schema
+from sqlalchemy import or_
+from sqlalchemy.orm.exc import NoResultFound
+import pytz
+from datetime import datetime
+import urllib.error
+
+from app.api.bootstrap import api
+from app.api.data_layers.EventCopyLayer import EventCopyLayer
+from app.api.helpers.db import safe_query
+from app.api.helpers.files import create_save_image_sizes
+from app.api.helpers.permission_manager import has_access
+from app.api.helpers.utilities import dasherize
+from app.api.schema.events_orga import EventOrgaSchemaPublic, EventOrgaSchema
+from app.models.role import Role
+
+# models
```

Comment:
Remove this comment and optimize the imports according to PEP8 guidelines. If using pycharm simply `ctrl+alt+o`.